Stp.Data/Entities/TestSection.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stp.Data.Entities
+{
+
```

Comment:
[Table(nameof(TestSection))] attribute is needed. Without this attribute, the table name generated according to the name of the corresponding DbContext field (TestSectionS)  - which is incorrect.